libs/community/langchain_community/graph_vectorstores/cassandra.py

```diff
@@ -12,16 +13,44 @@
 from langchain_core._api import beta
 from langchain_core.documents import Document
 from langchain_core.embeddings import Embeddings
-from langchain_core.graph_vectorstores.base import (
-    GraphVectorStore,
-    Node,
-    nodes_to_documents,
-)
+from langchain_core.graph_vectorstores.base import GraphVectorStore
+from langchain_core.graph_vectorstores.links import METADATA_LINKS_KEY, Link
 
 from langchain_community.utilities.cassandra import SetupMode
 
 if TYPE_CHECKING:
     from cassandra.cluster import Session
+    from ragstack_knowledge_store import Node
```

Comment:
does this need to be in the `try` import block below?

Reply:
You mean in `__init__` ?